src/map/party.cpp

```diff
@@ -754,15 +754,12 @@ void CParty::ReloadParty()
                     //    effects->AddMemberEffects(PChar);
 
                     // Inject the party leader's trusts into the party list
-                    CCharEntity* leader = (CCharEntity*)GetLeader();
-                    if (leader->PTrusts.size() > 0)
+                    CCharEntity* PLeader = (CCharEntity*)GetLeader();
```

Comment:
I think you need a nullptr check here.  If PLeader is in another zone that's on another server, I am fairly certain GetLeader will return nullptr.

If you wish to test this scenario, I can help you set up multiple servers on Discord (Lotus) if you need it

Reply:
No need to test this, you're absolutely right. Thanks for spotting this.

Reply:
Fixed. https://github.com/DarkstarProject/darkstar/pull/4729/commits/4c7fbcd279c84355896d96fa8952976015be8e01